gooddata-sdk/docs/services/catalog-data-source.rst

```diff
@@ -228,6 +228,24 @@ The *gooddata_sdk.catalog_data_source* supports the following action API calls:
 
     Invalidate cache of your computed reports to force your analytics to be recomputed.
 
+* ``scan_data_source(data_source_id: str, scan_request: CatalogScanModelRequest = CatalogScanModelRequest(), report_warnings: bool = False)``
+
+    Returns *CatalogScanResultPdm*.
+
+    Scan data source specified by its id and optionally by specified scan request. *CatalogScanResultPdm* contains PDM and warnings. Warnings contain information about columns, which were not added to the result PDM because their data types are not supported by GD.CN (e.g. ARRAY). Additional parameter report_warnings can be passed to suppress or to report warnings. By default warnings are returned but not reported to STDOUT. If you set report_warnings to True, warnings are reported to STDOUT.
```

Comment:
Scan data source specified by its id and optionally by specified scan request. *CatalogScanResultPdm* contains PDM and warnings. Warnings contain information about columns which were not added to the PDM because their data types are not supported. Additional parameter report_warnings can be passed to suppress or to report warnings. By default warnings are returned but not reported to STDOUT. If you set report_warnings to True, warnings are reported to STDOUT.